std/digest/sha.d

```diff
@@ -115,45 +115,16 @@ version (LittleEndian) import core.bitop : bswap;
 
 public import std.digest;
 
-/*
- * Helper methods for encoding the buffer.
- * Can be removed if the optimizer can inline the methods from std.bitmanip.
- */
-private ubyte[8] nativeToBigEndian(ulong val) @trusted pure nothrow @nogc
+version (LittleEndian)
 {
-    version (LittleEndian)
-        immutable ulong res = (cast(ulong)  bswap(cast(uint) val)) << 32 | bswap(cast(uint) (val >> 32));
```

Comment:
This is reinventing `core.bitmap.bswap()`

Reply:
Understandable because [this code was written](https://github.com/dlang/phobos/commit/9dc61f673446fe299c1064f39ab8f244099802ae) before [64-bit bswap was added to core.bitop](https://github.com/dlang/druntime/commit/d07e682f21b0043354fc4b9ca6d5d2b176a63539).

Reply:
https://github.com/dlang/phobos/pull/7534